skimage/feature/peak.py

```diff
@@ -105,16 +105,16 @@ def peak_local_max(image, min_distance=1, threshold_abs=None,
            [ 0. ,  0. ,  0. ,  0. ,  0. ,  0. ,  0. ],
            [ 0. ,  0. ,  0. ,  0. ,  0. ,  0. ,  0. ]])
 
-    >>> peak_local_max(img1, min_distance=1)
+    >>> peak_local_max(img1, min_distance=1)  # doctest: +SKIP
```

Comment:
Why all these skips?  For 32-bit?  Why not just skip the doctests on those platforms?